src/content/docs/new-relic-control/agent-control/authentication.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: "Authentication"
+metaDescription: "Overview of the Agent Control authentication process."
+freshnessValidatedDate: never
+---
+<Callout title="preview">
+  We're still working on this feature, but we'd love for you to try it out!
+
+  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
+</Callout>
+
+New Relic Fleet Control requires all requests coming from Agent Control to be signed, ensuring that the communication
+is authenticated and coming from a verified source.
+
+Agent Control must have a valid `L2 System Identity` and use it to sign all requests. L2 System Identities do not
+expire and can be used throughout the lifecycle of Agent Control.
```

Comment:
Can they be revoked/rotated?

For sure we recreate one if you delete the secret, but I am not sure whether you can marked as invalid an old one

Reply:
Good question 🤔 I know that it can be done through NerdGraph, but I'm not sure about the permissions required. I asked and tagged you in the question.